sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-servicebus/pom.xml

```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-messaging-servicebus</artifactId>
-      <version>7.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;current} -->
+      <version>7.0.0-beta.4</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;current} -->
```

Comment:
revert

Reply:
This particular change should be reverted. The tag is in error and I've notified the spring team that this should be dependency, not current.